cmd/sriov-network-config-daemon/start.go

```diff
@@ -81,6 +106,14 @@ func runStartCmd(cmd *cobra.Command, args []string) error {
 		startOpts.nodeName = name
 	}
 
+	// TODO(adrianc): move allowed disable plugins to globals pkg
+	allowedDisablePlugins := map[string]struct{}{"mellanox": {}}
```

Comment:
What about `deactivatablePlugins` or `activatablePlugins` if we switch to a positive instead (which is maybe better to avoid double negatives and have shorter names throughout).

Reply:
addressed, will upload PR soon